plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectPushdown.java

```diff
@@ -113,6 +115,31 @@ public static boolean isCompressionCodecSupported(InputFormat<?, ?> inputFormat,
         return false;
     }
 
+    public static boolean isUncompressed(InputFormat<?, ?> inputFormat, Path path)
```

Comment:
nit: It could be private, also usually we are trying to arrange methods by the order of invokations. If method A invokes method B method A should come first.